src/utils/DdoHelpers.ts

```diff
@@ -1,17 +1,12 @@
 import sha256 from 'crypto-js/sha256'
 import Web3 from 'web3'
-import LoggerInstance from './Logger'
 
-export function generateDid(erc721Address: string, chainId: number): string {
-  erc721Address = Web3.utils.toChecksumAddress(erc721Address)
-  const checksum = sha256(erc721Address + chainId.toString(10))
+export function generateDid(nftAddress: string, chainId: number): string {
+  nftAddress = Web3.utils.toChecksumAddress(nftAddress)
+  const checksum = sha256(nftAddress + chainId.toString(10))
   return `did:op:${checksum.toString()}`
 }
 
 export function getHash(data: any): string {
-  try {
-    return sha256(data).toString()
-  } catch (e) {
-    LoggerInstance.error('getHash error: ', e.message)
-  }
+  return sha256(data).toString()
```

Comment:
since this can fail think if the data send is not well formed, think we should keep the clause and have some error logs at least